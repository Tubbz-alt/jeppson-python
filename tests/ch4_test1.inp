   0.12      4.0     150.0       0.0102    0.1      1.217E-5
   0.10      3.0     100.0       0.0102    0.1      1.217E-5
   0.05      3.0     150.0       0.0102    0.1      1.217E-5
   0.05      3.0     100.0       0.0102    0.1      1.217E-5
   0.10      4.0     100.0       0.0102    0.1      1.217E-5
   0.05      3.0     141.0       0.0102    0.1      1.217E-5
