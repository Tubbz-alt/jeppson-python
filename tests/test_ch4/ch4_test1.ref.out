     0.12000     0.33333     0.10170     0.02293     1.89830    2.105782E+01
     0.10000     0.25000     0.07934     0.02481     1.92066    6.401589E+01
     0.05000     0.25000     0.12335     0.02207     1.87665    8.539451E+01
     0.05000     0.25000     0.12335     0.02207     1.87665    5.692967E+01
     0.10000     0.33333     0.11346     0.02234     1.88654    1.367725E+01
     0.05000     0.25000     0.12335     0.02207     1.87665    8.027084E+01
